src/main/java/com/book/controller/CartItemsController.java

```diff
@@ -28,6 +35,32 @@ public ResponseEntity<List<CartItemsEntity>> getCartItems(@RequestParam int id)
         }
 
         return new ResponseEntity<>(new ArrayList<CartItemsEntity>(), HttpStatus.OK);
+    }
+
+    @PostMapping("/addCartItems")
+    public ResponseEntity<String> addCartItems(@RequestBody CartItemsDTO cartItemsDTO) {
+        Optional<UsersEntity> usersEntity = userService.getUserByUserName(jwtUtils.getUserNameFromJwtToken(cartItemsDTO.getJwt()));
```

Comment:
this is an example of doing business logic in a controller. This logic should be executed in service layer